app/views/offers/_form.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= render "shared/error_messages", resource: offer %>
+
+<div class="flex flex-col space-y-3">
+  <% unless turbo_native_app? %>
+    <h2 class="text-xl font-bold leading-tight text-gray-900">
+      <%= t(".title") %>
+    </h2>
+  <% end %>
+
+  <%= form_with model: [conversation, offer] do |form| %>
+    <div class="my-4 flex flex-col">
+      <div class="space-y-6">
+        <div>
+          <%= form.label :start_date, class: "block text-sm font-medium text-gray-700" %>
+          <div class="mt-1">
+            <%= form.date_field :start_date, value: Date.current, class: field_classes(form, :start_date) %>
+          </div>
+        </div>
+
+        <div>
+          <%= form.label :pay_rate_value, class: "block text-sm font-medium text-gray-700" %>
+          <div class="relative mt-1 rounded-md shadow-sm">
+            <div class="pointer-events-none absolute inset-y-0 left-0 flex items-center pl-3">
+              <span class="text-gray-500 sm:text-sm">$</span>
+            </div>
+            <%= form.text_field :pay_rate_value, type: "number", value: 50, required: true, class: "pl-7 pr-20 #{field_classes(form, :pay_rate_value)}" %>
+            <div class="absolute inset-y-0 right-0 flex items-center">
+              <%= form.select :pay_rate_time_unit,
+                              Offer.pay_rate_time_units.map { |unit, value| [unit, value] },
+                              {},
```

Comment:
Simpler to use https://apidock.com/rails/v5.2.3/ActionView/Helpers/FormOptionsHelper/options_from_collection_for_select ?
```suggestion
                              options_from_collection_for_select(Offer.pay_rate_time_units. :first, :last)
```